src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/CompletionHandler.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.VisualStudio.LanguageServerClient.Razor.HtmlCSharp
     internal class CompletionHandler : IRequestHandler<CompletionParams, SumType<CompletionItem[], CompletionList>?>
     {
         private static readonly IReadOnlyList<string> RazorTriggerCharacters = new[] { "@" };
-        private static readonly IReadOnlyList<string> CSharpTriggerCharacters = new[] { " ", "(", "=", "#", ".", "<", "[", "{", "\"", "/", ":", ">", "~" };
+        private static readonly IReadOnlyList<string> CSharpTriggerCharacters = new[] { " ", "(", "#", ".", "<", "[", "{", "\"", "/", ">", "~" };
```

Comment:
I find it interesting that we use the same trigger characters as Roslyn but still end up with different completion experience for `=` compared to regular C# files. I can see it happen if we explicitly set the CompletionTriggerKind to `Invoked` instead of `TriggerCharacter` but I don't think we do that. Should we probe more to find the actual root cause?